common/lib/api-client/request-headers.js

```diff
@@ -3,13 +3,21 @@ const uuid = require('uuid')
 const {
   API: { VERSION },
   APP_VERSION,
+  CUSTOM_HEADERS,
 } = require('../../../config')
 
-module.exports = (format = 'application/vnd.api+json') => {
-  return {
+module.exports = (req, format = 'application/vnd.api+json') => {
```

Comment:
How many calls do we currently make that aren't `json`?

Reply:
The csv download. That's it